### PR TITLE
Feature/uweb 6068 nav

### PR DESCRIPTION
**Pull request by zwaardje**

### CODE REVIEW

File: src/js/components/organisms/page-footer/page-footer.vue.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            </locale-selector>
+        </div>
+
+        <div class="footer__tertiary">
+            <nav class="footer__legal-navigation" v-if="legalNavigation && legalNavigation.length">
+                <footer-navigation-category class="footer__legal-navigation__column" :label="copyrightLabel" :items="legalNavigation" />       
+            </nav>
+       </div>
+    </div>
+</footer>
```
Comment (matijs) [inline src/js/components/organisms/page-footer/page-footer.vue.html]: Your IDE/editor is lacking an EditorConfig plugin. There is a newline missing at the end of this file.

File: src/js/components/organisms/page-footer/page-footer.scss [hunk 2]
```diff
@@ -74,30 +68,32 @@
...
+    display: flex;
+    justify-content: center;
+
+    @include min-mobile-xl {
+        justify-content: flex-start;
+    }
 }
 
-.footer__legal {
+.footer__legal-navigation {
```
Comment (matijs) [inline src/js/components/organisms/page-footer/page-footer.scss]: Legal is more descriptive than `legal-nativation` imo…

File: src/js/components/molecules/footer-navigation-category/footer-navigation-category.vue.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="footer-navigation-category__column" v-if="items.length">
```
Comment (matijs) [inline src/js/components/molecules/footer-navigation-category/footer-navigation-category.vue.html]: `footer-navigation-category` or `footer-navigation__category`… `column` is too presentational.

File: src/js/components/molecules/footer-navigation-category/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export { FooterNavigationCategory as default } from './footer-navigation-category';
```
Comment (matijs) [inline src/js/components/molecules/footer-navigation-category/index.ts]: How do you feel about making `FooterNavigationCategory` a default export so you can `export { default } from './footer-navigation-category';`? here?

File: src/js/components/molecules/footer-navigation-category/stories/footer-navigation-category.html [hunk 5]
```diff
@@ -0,0 +1 @@
+<footer-navigation-category  v-bind="$props"></footer-navigation-category>
```
Comment (matijs) [inline src/js/components/molecules/footer-navigation-category/stories/footer-navigation-category.html]: Self-close

File: src/js/components/molecules/footer-navigation-category/footer-navigation-category.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+@WithRender
+@Component({
+    name: 'FooterNavigationCategory',
+})
+
+export class FooterNavigationCategory extends Vue implements FooterNavigationCategoryProps {
+    @Prop({ type: String, required: true }) label!: FooterNavigationCategoryProps['label'];
+    @Prop({ type: Array, required: false }) items?: FooterNavigationCategoryProps['items'];
+
+    splitCount: number = 8
```
Comment (matijs) [inline src/js/components/molecules/footer-navigation-category/footer-navigation-category.ts]: No need to put this inside the class.

File: src/js/components/molecules/footer-navigation-category/footer-navigation-category.ts [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+    classes() {
+        return this.items && this.items.length >= this.splitCount ? 'footer-navigation-category--columns' : '';
```
Comment (matijs) [inline src/js/components/molecules/footer-navigation-category/footer-navigation-category.ts]: `columns` → `multi-column`?

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.html [hunk 8]
```diff
@@ -0,0 +1 @@
+<main-navigation-item  v-bind="$props"></main-navigation-item>
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/stories/main-navigation-item.html]: self-close

File: src/js/components/molecules/main-navigation-item/data/main-navigation-item.data.ts [hunk 9]
```diff
@@ -0,0 +1,42 @@
+export const data = {
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/data/main-navigation-item.data.ts]: suggest using data from the live site

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts [hunk 10]
```diff
@@ -0,0 +1,5 @@
+export default {
+    label: '',
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts]: suggest alphabetising the properties

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.stories.ts [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+const getStory = type => () => {
+    const knobs = getKnobs(
+        data[type],
+        knobsFormat,
+    );
+
+    return {
+        props: {
+            label: { default: knobs.label },
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/stories/main-navigation-item.stories.ts]: suggest alphabetising the properties

File: src/js/components/molecules/main-navigation-item/main-navigation-item.ts [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+import BrowserCapabilities from 'utils/browser-capabilities';
+
+@WithRender
+@Component({
+    name: 'MainNavigationItem',
+})
+
+export class MainNavigationItem extends Vue implements MainNavigationItemProps {
+    @Prop({ type: Boolean, required: false }) mainNavOpen?: boolean;
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/main-navigation-item.ts]: `required: false` can be removed; as far as i know, its the default value for `required`

File: src/js/components/molecules/main-navigation-item/main-navigation-item.ts [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+
+    @Prop({ type: String, required: true }) label!: MainNavigationItemProps['label'];
+    @Prop({ type: String, required: true }) url!: MainNavigationItemProps['url'];
+    @Prop({ type: Array, required: false }) categories?: MainNavigationItemProps['categories'];
+
+    @Prop({ type: String, required: true }) itemId!: string;
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/main-navigation-item.ts]: don’t see this property in the storybook data file or in the model

File: src/js/components/molecules/main-navigation-item/main-navigation-item.ts [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+    @Prop({ type: Boolean, required: false }) isCompact?: boolean;
+    @Prop({ type: Boolean, required: true }) active!: boolean;
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/main-navigation-item.ts]: i don’t see this property in the storybook data file, so storybook shows a console error. maybe add it to the data file?

File: src/js/components/molecules/main-navigation-item/main-navigation-item.ts (hunk 12 above)
Comment (zwaardje) [reply src/js/components/molecules/main-navigation-item/main-navigation-item.ts]: I know, just to be more clear

File: src/js/components/molecules/footer-navigation-category/index.ts (hunk 4 above)
Comment (zwaardje) [reply src/js/components/molecules/footer-navigation-category/index.ts]: No changing, this is throughout the whole design system

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts (hunk 10 above)
Comment (zwaardje) [reply src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts]: Ide does not support that, maybe later

File: src/js/components/molecules/footer-navigation-category/footer-navigation-category.vue.html [hunk 15]
```diff
@@ -0,0 +1,8 @@
...
+    <h3 class="footer-navigation-category__title">{{ label }}</h3>
+    <ul class="footer-navigation-category__list" v-if="items && items.length">
+        <li v-for="link in items" class="footer-navigation-category__item">
+            <content-link class="footer-navigation-category__link link link--reversed" v-bind="link" />
+        <li>
```
Comment (dan-entous) [inline src/js/components/molecules/footer-navigation-category/footer-navigation-category.vue.html]:
```html
</li>
```

File: src/js/components/molecules/main-navigation-item/data/main-navigation-item.data.ts [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+    default: {
+        itemId: 1,
```
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/data/main-navigation-item.data.ts]:
```js
        itemId: '1',
```

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts (hunk 10 above)
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/stories/main-navigation-item.knobs-format.ts]:
```js
    active: false,
    categories: [],
    itemId: '',
    label: '',
    url: '',
```

File: src/js/components/molecules/main-navigation-item/stories/main-navigation-item.stories.ts (hunk 11 above)
Comment (dan-entous) [inline src/js/components/molecules/main-navigation-item/stories/main-navigation-item.stories.ts]:
```js
            active: { default: knobs.active },
            categories: { default: knobs.categories },
            itemId: { default: knobs.itemId },
            label: { default: knobs.label },
            url: { default: knobs.url },
```

**Pull request merged**